Typescript fails to compile the plugin

## 🐛 Bug Report

Application that uses `fastify-jwt@2.1.0` fails to compile with typescript.

## To Reproduce

Steps to reproduce the behavior:

1. Install and import the module.
2. Try to compile:
```sh
npx tsc
node_modules/fastify-jwt/jwt.d.ts:77:123 - error TS2694: Namespace '"node_modules/fastify/fastify"' has no exported member 'JWTTypes'.

...

Found 2 errors.
```

## Your Environment

- *node version*: v14.2.0
- *typescript version*: 3.9.5
- *fastify version*: 3.0.0-rc.3
- *os*: Mac

--- a/jwt.d.ts
+++ b/jwt.d.ts
@@ -41,13 +41,13 @@ declare module 'fastify' {
     jwt: JWT;
   }
 
-  interface FastifyReplyInterface {
+  interface FastifyReply {
     jwtSign(payload: JWTTypes.SignPayloadType, options?: jwt.SignOptions): Promise<string>;
     jwtSign(payload: JWTTypes.SignPayloadType, callback: JWTTypes.SignCallback): void;
     jwtSign(payload: JWTTypes.SignPayloadType, options: jwt.SignOptions, callback: JWTTypes.SignCallback): void;
   }
 
-  interface FastifyRequestInterface {
+  interface FastifyRequest {
     jwtVerify<Decoded extends JWTTypes.VerifyPayloadType>(options?: jwt.VerifyOptions): Promise<Decoded>;
     jwtVerify<Decoded extends JWTTypes.VerifyPayloadType>(callback: JWTTypes.VerifyCallback<Decoded>): void;
     jwtVerify<Decoded extends JWTTypes.VerifyPayloadType>(

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "steed": "^1.1.3"
   },
   "devDependencies": {
-    "fastify": "^3.0.0-rc.3",
+    "fastify": "^3.0.0-rc.4",
     "fastify-cookie": "^4.0.1",
     "standard": "^14.0.2",
     "tap": "^14.10.7",

